config/eslint.js

```diff
@@ -548,6 +549,131 @@ export default [
+      "unicorn/no-this-assignment": ["error"],
+      "unicorn/no-typeof-undefined": ["error"],
+      "unicorn/no-unnecessary-await": ["error"],
+      "unicorn/no-unnecessary-polyfills": ["error"],
+      "unicorn/no-unreadable-array-destructuring": ["error"],
+      "unicorn/no-unreadable-iife": ["error"],
+      "unicorn/no-unused-properties": ["error"],
+      "unicorn/no-useless-fallback-in-spread": ["error"],
+      "unicorn/no-useless-length-check": ["error"],
+      "unicorn/no-useless-promise-resolve-reject": ["error"],
+      "unicorn/no-useless-spread": ["error"],
+      "unicorn/no-useless-switch-case": ["error"],
+      "unicorn/no-useless-undefined": ["error"],
+      "unicorn/no-zero-fractions": ["error"],
+      "unicorn/number-literal-case": ["error"],
+      "unicorn/numeric-separators-style": ["error"],
+      "unicorn/prefer-add-event-listener": ["error"],
+      "unicorn/prefer-array-find": ["error"],
+      "unicorn/prefer-array-flat-map": ["error"],
+      "unicorn/prefer-array-flat": ["error"],
+      "unicorn/prefer-array-index-of": ["error"],
+      "unicorn/prefer-array-some": ["error"],
+      "unicorn/prefer-at": ["error"],
+      "unicorn/prefer-blob-reading-methods": ["error"],
+      "unicorn/prefer-code-point": ["error"],
+      "unicorn/prefer-date-now": ["error"],
+      "unicorn/prefer-default-parameters": ["error"],
+      "unicorn/prefer-dom-node-append": ["error"],
+      "unicorn/prefer-dom-node-dataset": ["error"],
+      "unicorn/prefer-dom-node-remove": ["error"],
+      "unicorn/prefer-dom-node-text-content": ["error"],
+      "unicorn/prefer-event-target": ["error"],
+      "unicorn/prefer-export-from": ["error"],
+      "unicorn/prefer-global-this": ["error"],
+      "unicorn/prefer-includes": ["error"],
+      "unicorn/prefer-json-parse-buffer": ["error"],
+      "unicorn/prefer-keyboard-event-key": ["error"],
+      "unicorn/prefer-logical-operator-over-ternary": ["error"],
+      "unicorn/prefer-math-min-max": ["error"],
+      "unicorn/prefer-math-trunc": ["error"],
+      "unicorn/prefer-modern-dom-apis": ["error"],
+      "unicorn/prefer-modern-math-apis": ["error"],
+      "unicorn/prefer-module": ["error"],
+      "unicorn/prefer-native-coercion-functions": ["error"],
+      "unicorn/prefer-negative-index": ["error"],
+      "unicorn/prefer-node-protocol": ["error"],
+      "unicorn/prefer-number-properties": ["error"],
+      "unicorn/prefer-object-from-entries": ["error"],
+      "unicorn/prefer-optional-catch-binding": ["error"],
+      "unicorn/prefer-prototype-methods": ["error"],
+      "unicorn/prefer-query-selector": ["error"],
+      "unicorn/prefer-reflect-apply": ["error"],
+      "unicorn/prefer-regexp-test": ["error"],
+      "unicorn/prefer-set-has": ["error"],
+      "unicorn/prefer-set-size": ["error"],
+      "unicorn/prefer-spread": ["off"],
+      "unicorn/prefer-string-raw": ["off"],
+      "unicorn/prefer-string-replace-all": ["error"],
```

Comment:
Unfortunately this is incompatible with Node.js v14.18.0, see:

https://github.com/ericcornelissen/shescape/actions/runs/12339815043/job/34436552265?pr=1812

This rule can be enabled and adopted when support for Node.js v14.18.0 is dropped.

Reply:
This has been resolved with 339da06e00ffccd59da571d55dbf7da3da52204a but an issue to enable this still needs to be created.

Reply:
Fully resolved with https://github.com/ericcornelissen/shescape/issues/1814